components/service/experiments/README.md

```diff
@@ -177,32 +177,36 @@ An individual experiment record looks e.g. like this:
+| schema                    | number |     x    | Timestamp of when the schema was last modified. This is provided by Kinto.                                                                      |                                                                 |
+| buckets                   | object |     x    | Object containing a bucket range to match users. Every user is in one of 1000 buckets (0-999).                                                  |                                                                 |
+|  buckets.start            | number |     x    | The minimum bucket to match.                                                                                                                    |                                                                 |
+| buckets.count             | number |     x    | The number of buckets to match from start. If (start + count >= 999), evaluation will wrap around.                                              |                                                                 |
+| branches                  | object |     x    | Object containing the parameters for ratios for randomized enrollment into different branches.                                                  |                                                                 |
+| branches[i].name          | string |     x    | The name of that branch.                                                                                                                        | `"control"` or `"red-button"`                                   |
+| branches[i].ratio         | number |     x    | The weight to randomly distribute enrolled clients among the branches.                                                                          |                                                                 |
+| match                     | object |     x    | Object containing the filter parameters to match specific user groups.                                                                          |                                                                 |
+| match.app_id              | regex  |          | The app ID (package name)                                                                                                                       | `"^org.mozilla.firefox_beta${'$'}|^org.mozilla.firefox${'$'}"`  |
+| match.device_manufacturer | regex  |          | The Android device manufacturer.                                                                                                                |                                                                 |
+| match.device_model        | regex  |          | The Android device model, [`Build.DEVICE`]. This is the codename for the device's industrial design.                                            | `"^(flame|coral)$"` (to match Pixel 4 or Pixel 4 XL)            |
```

Comment:
The pipe in between `flame|coral` is breaking the table.  Nothing shows up after this pipe.